pkg/config/config.go

```diff
@@ -84,8 +84,7 @@ const (
 type cgroupVersion uint8
 
 const (
-	noCgroupVersion cgroupVersion = iota
-	cgroupVersion1
+	cgroupVersion1 cgroupVersion = iota
```

Comment:
In `detectCgroupVersion` you return 0 to indicate there's no cgroup. But that's actually `cgroupVersion1`, so maybe you need `cgroupVersion1 cgroupVersion = iota + 1`

Reply:
Ahh yes!